src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -69,6 +70,19 @@ public void execute_validIndexFilteredList_success() {
         assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_deletePersonInActivity_throwsCommandException() {
+        showPersonAtIndex(model, INDEX_FIRST);
+        DeleteCommand deleteCommand = new DeleteCommand(INDEX_FIRST);
+        Person personToDelete = model.getFilteredPersonList().get(INDEX_FIRST.getZeroBased());
+        String errmsg = String.format(DeleteCommand.MESSAGE_PERSON_INVOLVED_ACTIVITY,
+                personToDelete.getName());
+
+        Model tempmodel = new ModelManager(
+                getTypicalAddressBook(), new UserPrefs(), new InternalState(), getTypicalActivityBook());
```

Comment:
I think this line should come first, then everything else executes off `tempmodel`?

Reply:
No, its fine. The command gets executed only on the `assertCommandFailure` step